runtime/src/lib.rs

```diff
@@ -202,7 +202,7 @@ macro_rules! create_zeitgeist_runtime {
                 Proxy: pallet_proxy::{Call, Event<T>, Pallet, Storage} = 33,
 
                 // Third-party
-                Currency: orml_currencies::{Call, Pallet, Storage} = 40,
+                AssetManager: orml_currencies::{Call, Pallet, Storage} = 40,
```

Comment:
In that case you are right, that it's always a CamelCase version of the pallet name. But I really like the name `AssetManager` and took so much thought into it. I think, that the pallet name `orml_currencies` is not really fitting, because I would tend to classify a token as an asset rather than a currency.

Reply:
No problem, if you feel strongly about this, we can keep the name. :wink:

I personally don't care about the distinction between currency and asset. I'd rather maintain the naming pattern for the pallets.